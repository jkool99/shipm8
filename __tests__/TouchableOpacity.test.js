import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import { Text, TouchableOpacity } from 'react-native';

describe('TouchableOpacity', () => {
  it('renders correctly', () => {
    const instance = renderer
      .create(
        <TouchableOpacity style={{}}>
          <Text>This should be a button</Text>
        </TouchableOpacity>,
      )
      .toJSON();

    expect(instance).toMatchSnapshot();
  });
});
