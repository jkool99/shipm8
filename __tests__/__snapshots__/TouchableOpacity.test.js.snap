// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TouchableOpacity renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <Text>
    This should be a button
  </Text>
</View>
`;
